### ISSUE TITLE
Is slate still under active development? 

### ISSUE THREAD

**Issue by logicbomb** [jigish/slate#379]
Issues are being opened but there hasn't been any commits in a while.   The tool is awesome, are updates planned?

**Comment by p0deje** [comment]
:+1: to know the answer

**Comment by jeberly** [comment]
i guess not....

**Comment by cenkozan** [comment]
You can pull the improvements you like, repackage it in Xcode, and overwrite Slate. That's what I've been doing lately.

**Comment by p0deje** [comment]
At least at would be great to have these improvements in a single repo, not in multiple forks.

**Comment by dsummersl** [comment]
amen - and more specifically, I'd be willing to spend a little time to help. Anybody know the fixes worth merging (all outstanding requests?).

**Comment by jevy** [comment]
+1

**Comment by p0deje** [comment]
@dsummersl I guess so. Looking forward to see your fork!

**Comment by sdegutis** [comment]
I [forked it](https://github.com/sdegutis/phoenix) (actually a fork of my original Slate fork), improved the performance, rewrote the JavaScript bridge, removed some APIs, and simplified some other APIs.

**Comment by sebald** [comment]
@sdegutis does Phoenix also support the display detection like slate?

**Comment by sdegutis** [comment]
@sebald It has functions to know what screen (aka. display or monitor) a given window is on, and to move it to a given screen. Is that what you mean?

**Comment by sdegutis** [comment]
By the way @sebald, I just added documentation for Phoenix, which might help answer your question:

https://github.com/sdegutis/Phoenix/wiki/JavaScript-API-documentation

**Comment by sebald** [comment]
@sdegutis  Thx for the fast reply and the docu :) really appeaciate it.

I am actually just looking for something that restores my windows to its previous or a predefined state depending on the monitor they are. It super annoying when I unplug my thunderbolt display...

**Comment by harizvi** [comment]
Should I expect my current .slate.js to work as is with Phoenix, or will I need various changes?

**Comment by sdegutis** [comment]
@harizvi No, the API has been changed to enable performance increases and clean documentation enough to the point where it is not backwards compatible anymore. However, it should be a straightforward-enough API to be able to rewrite your config in a few minutes. Just be sure to check the docs to make sure it has all the functionality you need! Some Slate features were dropped in my fork, although there are plans to provide some of them again in the future using a new API.

**Comment by logicbomb** [comment]
Good work!  What are your plans w/r/t spaces support - specifically sending
windows to their own space? And moving windows from spaces by hotkey?

**Comment by sdegutis** [comment]
@logicbomb See this issue: https://github.com/sdegutis/Phoenix/issues/6

**Comment by keith** [comment]
:+1: for [Phoenix](https://github.com/sdegutis/Phoenix) slightly different config, haven't seen any issues yet[*](https://github.com/sdegutis/Phoenix/issues/11).

**Comment by sdegutis** [comment]
@Keithbsmiley Well, except the one you filed ;)

**Comment by sdegutis** [comment]
@Keithbsmiley Thanks for putting your config file [here in your repo](https://github.com/Keithbsmiley/dotfiles/blob/496c1009c1f8cc28d53f40d40abf4060a69cedb5/phoenix.js) for us to read and steal from :)

**Comment by keith** [comment]
:+1: ported from my [slate config](https://github.com/Keithbsmiley/dotfiles/blob/master/slate.js)

**Comment by norswap** [comment]
Does someone has a Phoenix translation of the default slate config?
Also, is there a default Phoenix config somewhere?

**Comment by keith** [comment]
Here's my [phoenix conf](https://github.com/Keithbsmiley/dotfiles/blob/master/phoenix.js) from my [slate conf](https://github.com/Keithbsmiley/dotfiles/blob/master/slate.js)

**Comment by keith** [comment]
Here's [a list of example configs](https://github.com/sdegutis/Phoenix/wiki)

**Comment by mattes** [comment]
so slate is dead? phoenix is the new slate?

**Comment by dsummersl** [comment]
Wel...to be clear - slate still works fine. I switched to phoenix and now pheonix itself is looking for a maintainer. So.

**Comment by keith** [comment]
I switched away from slate when it started freezing on a regular basis. Phoenix has been a lot more stable but YMMV. Also I'm planning on helping out with that open developer position. https://github.com/sdegutis/Phoenix/issues/28

**Comment by drawxcards** [comment]
@cenkozan do you mind provide the steps for repackaging in xcode? kinda new in xcode here..Also I install slate via homebrew, does it matter at all?

**Comment by cenkozan** [comment]
@gddabe You can use this http://stackoverflow.com/a/5579458 link to do what
you want.

**Comment by drawxcards** [comment]
@cenkozan thanks a lot, what I need to merge by myself is to fix the wrong name when taking snapshot jigish#393 . Now I've done it successfully.

**Comment by OleMchls** [comment]
:+1: [Phoenix](https://github.com/sdegutis/Phoenix). But anyway, too bad that this kind of cool projects just got abandoned

**Comment by chdiza** [comment]
I find that [Mjolnir](https://github.com/mjolnir-io/mjolnir), the successor to Hydra and Phoenix, is now usable at least for moving windows around and running shell commands from a hotkey.

I really hope that Slate is not dead, because I'd prefer to stick with something that still serves me so well.  But I'm preemptively exploring Mjolnir and getting my Slate config ported just in case.

Know that Mjolnir is (will be) much more than just a window-manipulator, but that if you don't want those extra features, you won't have to "install" them.

**Comment by chdiza** [comment]
> getting my Slate config ported

"Ported" is really far from the right word here, sorry.  I meant, "write in lua enough stuff so that Mjolnir can do all the things my Slate config enabled me to do".

**Comment by mattr-** [comment]
I'm working on bringing Slate back to life, but it's slow-going since I'm not a hardcore Mac dev (yet?). I'll probably rename the app, and some of those other sorts of things, but if you'd like, you can follow along at [mattr-/slate](https://github.com/mattr-/slate)

**Comment by sdegutis** [comment]
:+1:

**Comment by shrwnsan** [comment]
:+1:
